gcp/src/test/java/org/apache/iceberg/gcp/gcs/GCSOutputStreamTest.java

```diff
@@ -57,14 +58,16 @@ public void testWrite() {
 
   @Test
   public void testMultipleClose() throws IOException {
-    GCSOutputStream stream = new GCSOutputStream(storage, randomBlobId(), properties);
+    GCSOutputStream stream = new GCSOutputStream(storage, randomBlobId(), properties, MetricsContext
+        .nullMetrics());
```

Comment:
Nit: Avoid breaking up a chained call as an argument across two lines. In this case I would move all arguments to the next line.

But having `MetricsContext` and then `.nullMetrics()` on the next line is confusing.

Reply:
Thanks, I have made the changes.